egs/librimix/ConvTasNet/eval.py

```diff
@@ -18,6 +19,17 @@
 from asteroid.utils import tensors_to_device
 
 
+def import_wer():
+    try:
+        from jiwer import wer
```

Comment:
Since you're here @sw005320, what do you think about `jiwer`?
BTW, feel free to add any comment, advice etc..

Reply:
I checked the library and it looks good to me.
I was concerning punctuation handling, upper/lower case, and non-linguistic special annotations (e.g., <NOISE>), but the tool seems to handle everything.

Reply:
Again, thank a lot ! It's going to make it much easier to use jiwer directly rather than a non-python alternative for us.